CreateDeliveryEmployee.sql

```diff
@@ -0,0 +1,15 @@
+DELIMITER $$
+DROP PROCEDURE IF EXISTS create_deliveryEmployee $$
+CREATE PROCEDURE create_deliveryEmployee()
+BEGIN
+    START TRANSACTION;
+		CREATE TABLE Kainoos_AdamU.DeliveryEmployee (
```

Comment:
SalesEmployee in the other PR doesn't have a direct link to the BankInfo but the Delivery Employee does, could we keep it consistent